code/modules/mob/living/simple_animal/abnormality/he/road_home.dm

```diff
@@ -0,0 +1,381 @@
+/mob/living/simple_animal/hostile/abnormality/road_home
+	name = "The Road Home"
+	desc = "An abnormality ressembling a small girl."
+	icon = 'ModularTegustation/Teguicons/32x32.dmi'
+	icon_state = "road_home"
+	icon_living = "road_home"
+	maxHealth = 1300
+	health = 1300
+	move_resist = MOVE_FORCE_STRONG //So she can't be yeeted away and delayed indefinitely
+	density = FALSE //This means bodyblocking her isn't an option either.
```

Comment:
This will prevent projectiles from hitting it and let people move through.
You might want to edit the proc for movement instead; See how it's done with MoSB.

Reply:
I changed her density to true and she uses the proc now. Although it can look a little awkward it's still functional from what I've tested. I've also fixed a bug where she would move twice as fast as she's supposed to.